index.html

```diff
@@ -499,10 +499,10 @@
 
     ##### Grabbing a Frame for Post-Processing
     <pre class='example'>
-    navigator.mediaDevices.getUserMedia({video: true}).then(gotMedia, failedToGetMedia);
+    MediaDevices.getUserMedia({video: true}).then(gotMedia, failedToGetMedia);
```

Comment:
Apologies for missing this, but this is wrong. MediaDevices is the class variable - you can't call MediaDevices.getUserMedia since it's not a static function.

You have to call navigator.mediaDevcies.getUserMedia(), or it won't run.
(If I'm wrong, and you've actually seen code like this running, please provide the example!)